hugegraph-api/src/main/java/com/baidu/hugegraph/api/profile/ProfileAPI.java

```diff
@@ -59,6 +59,7 @@ public class ProfileAPI {
     private static String API_PROFILES = null;
 
     @GET
+    @Path("profiles")
```

Comment:
can we fix this issue and keep the origin url path?

Reply:
It cause by this PR #1500 

[this line](https://github.com/apache/incubator-hugegraph/commit/9ee9fe12f3349a54bbceb161d002200d3689863c#diff-76994bedad9c0d52982ee900de571f9db327acfd5a8ab902c606cc20dc62a19aR284)

Reply:
Modify the check rule or just change routing to "/profiles"?

Reply:
Thanks for pointing out the root cause.
IMO we cam add url "/" to WHITE_API_LIST to keep the continuity.

Reply:
I didn't notice that ProfileApi would use an blank url. So in order to keep the original url, I chose to delete the blank url check. #1893

Reply:
OK, let's keep the original url